auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -70,7 +70,7 @@
                 outlined dark large
                 v-bind="attrs"
                 v-on="on"
-                @click="startNumberedCompany()"
+                @click="startNumberedCompany(CorpTypes.BENEFIT_COMPANY)"
```

Comment:
@lambert-alex I assume what you're developing will call the same method but with different corp types. So this will be ready for you.

Reply:
Yes I am thanks for the heads up!